pages/docs/reference/delegated-properties.md

```diff
@@ -346,17 +385,20 @@ class C {
 
 </div>
 
-The Kotlin compiler provides all the necessary information about `prop` in the arguments: the first argument `this` refers to an instance of the outer class `C` and `this::prop` is a reflection object of the `KProperty` type describing `prop` itself.
+The Kotlin compiler provides all the necessary information about `prop` in the arguments: the first argument `this` refers
+to an instance of the outer class `C` and `this::prop` is a reflection object of the `KProperty` type describing `prop` itself.
 
-Note that the syntax `this::prop` to refer a [bound callable reference](reflection.html#bound-function-and-property-references-since-11) in the code directly is available only since Kotlin 1.1.  
+Note that the syntax `this::prop` to refer a [bound callable reference](reflection.html#bound-function-and-property-references-since-11)
+in the code directly is available only since Kotlin 1.1.  
 
 ### Providing a delegate (since 1.1)
 
-By defining the `provideDelegate` operator you can extend the logic of creating the object to which the property implementation is delegated.
-If the object used on the right hand side of `by` defines `provideDelegate` as a member or extension function, that function will be
-called to create the property delegate instance.
+By defining the `provideDelegate` operator you can extend the logic of creating the object to which the property implementation
+is delegated. If the object used on the right-hand side of `by` defines `provideDelegate` as a member or extension function,
+that function will be called to create the property delegate instance.
 
-One of the possible use cases of `provideDelegate` is to check property consistency when the property is created, not only in its getter or setter.
+One of the possible use cases of `provideDelegate` is to check property consistency when the property is created,
+not only in its getter or setter.
```

Comment:
Minor: the first part is about 'when' and the opposite is about 'where'. Perhaps, makes sense to reword both parts to be about the same category.
Also "property is created" sounds vague, because properties are not objects themselves.